neural_compressor/onnxrt/quantization/config.py

```diff
@@ -144,6 +144,11 @@ def get_model_info(model: Union[onnx.ModelProto, Path, str]) -> List[Tuple[str,
         logger.debug(f"Get model info: {filter_result}")
         return filter_result
 
+    @classmethod
+    def get_config_set_for_tuning(cls) -> Union[None, "RTNConfig", List["RTNConfig"]]:  # pragma: no cover

```

Comment:
@mengniwang95  @yuwenzho, we need to update it after the ORT enables the auto-tune.